app/src/webkit/java/org/mozilla/focus/webkit/WebkitView.java

```diff
@@ -65,6 +65,7 @@ public WebkitView(Context context, AttributeSet attrs) {
 
         linkHandler = new LinkHandler(this);
         setOnLongClickListener(linkHandler);
+        CookieManager.getInstance().setAcceptThirdPartyCookies(this, true);
```

Comment:
If this API is involved since API 21, should we add Api level checking?
```java
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
   ....
}
```

And `WebViewProvider.configureDefaultSettings` would be better place to do our webview-configuration.